src/js/input-range/input-range.jsx

```diff
@@ -11,6 +11,18 @@ import Track from './track';
 import { captialize, distanceTo, isDefined, isObject, length } from '../utils';
 import { DOWN_ARROW, LEFT_ARROW, RIGHT_ARROW, UP_ARROW } from './key-codes';
 
+function minimumRangeSizeValidator(props, propName, ...args) {
+  if (props[propName]) {
+    if (props[propName] < 0) {
+      return new Error('Minimum range size must be greater than or equal to 0');
+    }
+    if (props[propName] % props.step !== 0) {
+      return new Error('Minimum range size must be a multiple of the step size');
+    }
+  }
+  return PropTypes.number.apply(this, props, propName, args);
```

Comment:
looks like `checkPropTypes` is for checking in production. There's a paragraph just a bit further down on the page you linked that shows how we should handle this. 
> You might also see this error if you’re calling a PropTypes validator from your own custom PropTypes validator. In this case, the fix is to make sure that you are passing all of the arguments to the inner function. There is a more in-depth explanation of how to fix it on [this page](https://facebook.github.io/react/warnings/dont-call-proptypes.html#fixing-the-false-positive-in-third-party-proptypes).